gnucash/gtkbuilder/dialog-commodity.glade

```diff
@@ -305,7 +305,8 @@
                 <property name="model">liststore2</property>
                 <property name="has-entry">True</property>
                 <property name="entry-text-column">0</property>
-                <signal name="changed" handler="gnc_ui_commodity_changed_cb" swapped="no"/>
+                <property name="has-tooltip">True</property>
+                <property name="tooltip-markup" translatable="yes">Select a category for the commodity or enter a new one. One might use investment categories like STOCKS and BONDS or exchange names like NASDAQ and LSE.</property>                <signal name="changed" handler="gnc_ui_commodity_changed_cb" swapped="no"/>
```

Comment:
You lost a newline here. Did that happen while using the glade editor or did you just change the file via a text editor ?

Reply:
Fixed. I use emacs and avoid Glade.